eng/tools/mgmtreport/codecoverage.go

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+package main
```

Comment:
This file only contains one functions. If you want to reserve it, I think you could move all devops related function into this file and rename it to devops.go.